### DISCUSSION TITLE
Instagram в России

### DISCUSSION THREAD

**Discussion by dovecoteescapee** [dovecoteescapee/ByeDPIAndroid#63] [General]
Тема для обсуждения обхода блокировки Youtube.

В отличие от большинства сервисов, у instagram также заблокированы многие ip, потому запутывание DPI не помогает.

Предыдущие обсуждения:
#29 #55

**Comment by Archi98** [comment]
У меня инстаграм на адроиде обходит на ура, Твиттер с зависаниями тоже, однако проблемы с обходом ютюба.

С установкой версии ByeDPI на андроид версии 1.0.2-rc1 доступ к ютюбу пропал полностью. С предыдущей версией дела обстоят получше.

**Reply by dovecoteescapee** [reply]
> 1.0.2-rc1 доступ к ютюбу пропал полностью

Сохраняется ли проблема при Desync method: None? Должно быть как при отключённом приложении
